src/main/java/net/apnic/rdap/authority/config/AuthorityConfiguration.java

```diff
@@ -221,20 +232,15 @@ private void setupAuthorityStore()
 
             if(aConfig.getServers() != null)
             {
-                List<URI> servers = aConfig.getServers().stream()
-                    .map((String strURI) ->
-                    {
-                        try
-                        {
-                            return new URI(strURI);
-                        }
-                        catch(URISyntaxException ex)
-                        {
-                            throw new RuntimeException(ex);
-                        }
-                    })
-                    .collect(Collectors.toList());
-                authority.addServers(servers);
+                authority.addServers(
+                    aConfig.getServers().stream()
+                    .map(URI::create)
+                    .collect(Collectors.toList()));
+            }
+
+            if(aConfig.getDefaultServer() != null)
+            {
+                authority.setDefaultServer(URI.create(aConfig.getDefaultServer()));
             }
 
             authorityStore().addAuthority(authority);
```

Comment:
{suggestion} You could add a 'verify' method to the RDAPAuthority (which does the default/server uri checks, etc), and call it in the addAuthority method of the store. Then you could allow RDAPAuthority objects to be in an inconsistent state, but have it as a 'rule' of the store that any RDAPAuthorities it adds/retrieves are 'consistent'.